src/test/java/htsjdk/samtools/CRAMComplianceTest.java

```diff
@@ -362,6 +370,41 @@ public void testBAMThroughCRAMRoundTripViaPath() throws IOException {
         }
     }
 
+    @Test(description = "TC and TN should not be present in the output CRAM file")
+    public void testObsoleteDataSeriesNotWritten() throws IOException {
+
+        // 1301_slice_aux.cram has legacy dataseries TC, TN present in it.
+        // TC, TN are now removed and are no longer written by htsjdk
+        // This file is used to test if CRAM reader can ignore these dataseries.
+        // 1301_slice_aux.cram is taken from hts-specs repo. It uses reference files, ce.fa and ce.fa.fai.
+        // 1301_slice_aux.cram file: https://github.com/samtools/hts-specs/blob/master/test/cram/3.0/passed/1301_slice_aux.cram
+        // Reference files: https://github.com/samtools/hts-specs/tree/master/test/cram
```

Comment:
I think the first, third, fourth, and last lines of this comment are not really relevant/applicable to this test, since this is testing writing (so the structure of the input file is irrelevant - any CRAM, BAM or SAM input file would have worked just as well), and the reference is embedded right in the repo now, so this can be reduced to:

```suggestion
        // TC, TN are now removed and are no longer written by htsjdk
        // 1301_slice_aux.cram is taken from hts-specs repo. It uses reference files, ce.fa and ce.fa.fai.
        // 1301_slice_aux.cram file: https://github.com/samtools/hts-specs/blob/master/test/cram/3.0/passed/1301_slice_aux.cram
```